File: src/main/java/leets/leenk/domain/birthday/application/dto/response/UpcomingBirthdayUsersResponse.java
package leets.leenk.domain.birthday.application.dto.response;

import com.fasterxml.jackson.annotation.JsonInclude;
import io.swagger.v3.oas.annotations.media.Schema;
import lombok.Builder;

import java.util.List;

@Builder
@JsonInclude(JsonInclude.Include.NON_NULL)
public record UpcomingBirthdayUsersResponse(
        @Schema(description = "생일일 유저들")
        List<UpcomingBirthdayUserResponse> users
) {
}

File: src/main/kotlin/leets/leenk/domain/birthday/application/dto/response/UpcomingBirthdayUsersResponse.kt
package leets.leenk.domain.birthday.application.dto.response

import com.fasterxml.jackson.annotation.JsonInclude
import io.swagger.v3.oas.annotations.media.Schema

@JsonInclude(JsonInclude.Include.NON_NULL)
data class UpcomingBirthdayUsersResponse(
    @field:Schema(description = "생일일 유저들")
    val users: List<UpcomingBirthdayUserResponse>,
)
